docs/how-it-works/payjoin-v2-bip-77.md

```diff
@@ -1,24 +1,24 @@
 ---
 sidebar_position: 4
-title: Payjoin V2 (BIP 77)
-description: Serverless, asynchronous payjoin
+title: Async Payjoin (BIP 77, V2)
+description: Serverless, Asynchronous Payjoin
 ---
 
-:::note
+:::info
```

Comment:
I considered making this a [tip](https://docusaurus.io/docs/markdown-features/admonitions) because green is the color of "good", whereas info is blue and reused elsewhere, but I'm not sure if this qualifies as a _tip_ per se, and that keyword would be listed in the warning if I used that (right now it's "info").